.gitignore

```diff
@@ -30,3 +30,6 @@ docs/
 
 # Debug Logs
 debug.log
+
+# Endpoints with secrets
+assets/config/endpoints.json
```

Comment:
If this file hosts secrets, it's better to keep them outside the repository, in a different directory. This can still give rise to difficult situations.